CMakeLists.txt

```diff
@@ -1,479 +1,317 @@
-      JRL_CMAKE_MODULES
-      TARGET jrl-cmakemodules::jrl-cmakemodules
-      PROPERTY INTERFACE_INCLUDE_DIRECTORIES)
-    message(STATUS "JRL cmakemodules found on system at ${JRL_CMAKE_MODULES}")
-  else()
-    message(STATUS "JRL cmakemodules not found. Let's fetch it.")
-    include(FetchContent)
-    FetchContent_Declare(
-      "jrl-cmakemodules"
-      GIT_REPOSITORY "https://github.com/jrl-umi3218/jrl-cmakemodules.git")
-    FetchContent_MakeAvailable("jrl-cmakemodules")
-    FetchContent_GetProperties("jrl-cmakemodules" SOURCE_DIR JRL_CMAKE_MODULES)
-  endif()
-endif()
-
-function(set_standard_output_directory target)
-  set_target_properties(
-    ${target}
-    PROPERTIES RUNTIME_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/bin
-               LIBRARY_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/lib
-               ARCHIVE_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/lib)
-endfunction()
-
-# Disable -Werror on Unix for now.
-set(CXX_DISABLE_WERROR True)
-set(CMAKE_VERBOSE_MAKEFILE True)
-
-# ----------------------------------------------------
-# --- OPTIONS  ---------------------------------------
-# Need to be set before including base.cmake
-# ----------------------------------------------------
-option(INSTALL_DOCUMENTATION "Generate and install the documentation" OFF)
-option(SUFFIX_SO_VERSION "Suffix library name with its version" OFF)
-option(BUILD_TESTING_SCIPY
-       "Build the SciPy tests (scipy should be installed on the machine)" ON)
-
-# ----------------------------------------------------
-# --- Policy -----------------------------------------
-# CMake Policy setup
-# ----------------------------------------------------
-# Policy can be removed when cmake_minimum_required is updated.
-
-# We also set CMAKE_POLICY_DEFAULT_CMPXXXX because CMake modules can reset
-# policy and redefine some macros like `find_dependency` that will not use our
-# policy.
-
-# Use BoostConfig module distributed by boost library instead of using FindBoost
-# module distributed by CMake (to remove in 3.30).
-if(POLICY CMP0167)
-  cmake_policy(SET CMP0167 NEW)
-  set(CMAKE_POLICY_DEFAULT_CMP0167 NEW)
-endif()
-# install() DESTINATION paths are normalized (to remove in 3.31).
-if(POLICY CMP0177)
-  cmake_policy(SET CMP0177 NEW)
-  set(CMAKE_POLICY_DEFAULT_CMP0177 NEW)
-endif()
-include("${JRL_CMAKE_MODULES}/base.cmake")
-compute_project_args(PROJECT_ARGS LANGUAGES CXX)
-project(${PROJECT_NAME} ${PROJECT_ARGS})
-
-include("${JRL_CMAKE_MODULES}/boost.cmake")
-include("${JRL_CMAKE_MODULES}/python.cmake")
-include("${JRL_CMAKE_MODULES}/ide.cmake")
-include("${JRL_CMAKE_MODULES}/apple.cmake")
+include(cmake/get-jrl-cmakemodules.cmake)
 
-option(GENERATE_PYTHON_STUBS
-       "Generate the Python stubs associated to the Python library" OFF)
+jrl_configure_defaults()
 
-option(BUILD_WITH_CHOLMOD_SUPPORT "Build EigenPy with the Cholmod support" OFF)
+jrl_option(
+  GENERATE_PYTHON_STUBS
+  "Generate the Python stubs associated to the Python library"
+  OFF
+)
```

Comment:
This option is not used. Stub are always generated.
This is an issue when we cross build (stub as to be imported, but it's impossible when we are cross building)